underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -126,6 +126,22 @@ public GCSUnderFileSystem(AlluxioURI uri) throws ServiceException {
     mBucketMode = GCSUtils.translateBucketAcl(acl, mAccountOwnerId);
   }
 
+  protected GCSUnderFileSystem(AlluxioURI uri,
+                            GoogleStorageService googleStorageService,
+                            String bucketName,
+                            String bucketPrefix,
+                            short bucketMode,
+                            String accountOwner,
+                            String accountOwnerId) {
```

Comment:
Could you update this to be consistent with our style (arguments on one line, new line with 4-indent when hitting 100 char limit)?